app/mailers/progress_report_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class ProgressReportMailer < ApplicationMailer
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.progress_report_mailer.updated.subject
+  #
+  def updated(progress_report)
+    return if progress_report.manager_email.blank?
+
+    @indicator = progress_report.indicator
+    @manager_name = progress_report.manager_name
```

Comment:
@tmfrnz are these correct?

Reply:
see comment below: the manager_name should be taken from the category